set.mm

```diff
@@ -519565,6 +519862,356 @@ The sumset operation can be used for both group (additive) operations and
+      $d .x. c d g h i j k l m r t s u v w x y $.  $d C u x y $.  $d D x y $.
+      $d F c d g h i j k l m r s t u v w x y $.  $d H c d r $.  $d H g u v w $.
+      $d H j $.  $d H k l m $.  $d H l t $.  $d H p $.  $d H s $.
+      $d K k l m $.  $d K l r $.  $d K l t $.  $d K l u w $.  $d K p $.
+      $d M g l u $.  $d M h $.  $d M i $.  $d M j $.  $d M k l m $.
+      $d M l r $.  $d M l t $.  $d N l u w $.  $d N p $.  $d P g l u $.
+      $d P h $.  $d P i $.  $d P j $.  $d P k l m $.  $d P l r $.  $d P l t $.
+      $d P p $.  $d P q $.  $d Q l r $.  $d Q l t $.  $d Q s $.  $d Q u v w $.
+      $d R g l u $.  $d R i $.  $d R j $.  $d R k l m $.  $d R l r $.
+      $d R l t $.  $d R p $.  $d R q $.  $d S c d $.  $d S g l u w $.
+      $d S k l t $.  $d T l u w $.  $d U c d h $.  $d U g u v w $.  $d U i $.
+      $d U j $.  $d U k l m $.  $d U l r $.  $d U l t $.  $d U s $.
+      $d c d f $.  $d g p $.  $d i k $.  $d k l p $.  $d k l r $.  $d l ph $.
+      $d H g u v w $.  $d H i k $.  $d H j $.  $d H l t $.  $d H s $.
+      $d H x y $.  $d K l t $.  $d K x y $.  $d M g l t u $.  $d M i k $.
+      $d M j $.  $d M x y $.  $d N j $.  $d N l $.  $d N x y $.
+      $d P g l t u $.  $d P i k $.  $d P j $.  $d P q $.  $d P x y $.
+      $d Q g u v w $.  $d Q i k $.  $d Q j $.  $d Q l t $.  $d Q s $.
+      $d Q x y $.  $d R g l t u $.  $d R i k $.  $d R j $.  $d R q $.
+      $d R x y $.  $d S g l u w $.  $d S k l $.  $d S x y $.  $d T l $.
+      $d T x y $.  $d U g u v w $.  $d U i k $.  $d U j $.  $d U l t $.
+      $d U s $.  $d U x y $.  $d i p $.  $d j k $.  $d j ph $.  $d k x y $.
+      $d l m $.  $d l p $.  $d l r $.  $d ph q $.  $d ph t $.  $d ph x y $.
+      $d t w $.
+      1arithidomlem.1 $e |- ( ph -> Q e. P ) $.
+      1arithidomlem.2 $e |- ( ph -> A. g e. Word P (
+         E. k e. U ( M gsum F ) = ( k .x. ( M gsum g ) )
+           -> E. w E. u e. ( U ^m ( 0 ..^ ( # ` F ) ) ) (
+               w : ( 0 ..^ ( # ` F ) ) -1-1-onto-> ( 0 ..^ ( # ` F ) )
+             /\ g = ( u oF .x. ( F o. w ) ) ) ) ) $.
+      1arithidomlem.3 $e |- ( ph -> H e. Word P ) $.
+      1arithidomlem.4 $e |- ( ph -> E. k e. U ( M gsum ( F ++ <" Q "> ) )
+                                   = ( k .x. ( M gsum H ) ) ) $.
+      1arithidomlem.5 $e |- ( ph -> K e. ( 0 ..^ ( # ` H ) ) ) $.
+      1arithidomlem.6 $e |- ( ph -> Q ( ||r ` R ) ( H ` K ) ) $.
+      1arithidomlem.7 $e |- ( ph -> T e. U ) $.
+      1arithidomlem.8 $e |- ( ph -> ( T .x. Q ) = ( H ` K ) ) $.
+      1arithidomlem.9 $e |- ( ph
+                -> S : ( 0 ..^ ( # ` H ) ) -1-1-onto-> ( 0 ..^ ( # ` H ) ) ) $.
+      1arithidomlem.10 $e |- ( ph -> ( H o. S )
+          = ( ( ( H o. S ) prefix ( ( # ` H ) - 1 ) ) ++ <" ( H ` K ) "> ) ) $.
+      1arithidomlem.11 $e |- ( ph -> N e. U ) $.
+      1arithidomlem.12 $e |- ( ph -> ( M gsum ( F ++ <" Q "> ) )
+                                    = ( N .x. ( M gsum H ) ) ) $.
+      $( Lemma for 1arithidom.  (Contributed by Thierry Arnoux,
```

Comment:
"Lemma for 1arithidom." => "Lemma for ~ 1arithidom ." (I thought I remarked this already before, but maybe did not press the save button...)

But this is nonblocking, can be adjusted later.